docs/src/connecting/external-node.md

```diff
@@ -5,19 +5,25 @@ We can interact with the `Testnet` node by using the following example.
 ```rust,ignore
 {{#include ../../../examples/providers/src/lib.rs:connect_to_testnet}}
 ```
+>
+> For detailed information about various testnet networks and their optimal toolchain configurations for your project, please visit the following link:
+>
+> [networks](https://fuelbook.fuel.network/master/networks/networks.html)
```

Comment:
```suggestion
> **Note:** 
This network will only work with the versions of the toolchain specified [here](https://fuelbook.fuel.network/master/networks/networks.html).
> It will **not** work with versions other than the ones listed above. 
```

Reply:
There are certain compatible versions that are not included in the list provided on the Fuel Network's website at https://fuelbook.fuel.network/master/networks/networks.html.

Reply:
Can you please just write a sentence that explains what the user should do in order to have something that works.

Reply:
Isn't this part enough:`... their optimal toolchain configurations for your project , please visit the following link`.
If user clicks on the link, he will find information about all testnets and the recommended crate versions to use.
Versions specified on the websites are correct.

Reply:
I agree that it does say the truth, only we should double check the user understands that it won't work unless she uses the versions specified. Anyway I feel like I'm nitpicking on wording lol.